lib/svg-sprite/config.js

```diff
@@ -13,12 +13,13 @@ const fs = require('fs');
 const path = require('path');
 const yaml = require('js-yaml');
 const winston = require('winston');
+const Logger = require('winston/lib/winston/logger.js');
```

Comment:
No. But official exports has no Logger constructor. This behavior changed between v2 and v3 of package

Reply:
Hmm, unsure how proceed. It's just that this is fragile. Isn't there any other way to check for this?

Reply:
I prefer to drop this checking of being instance of winston.Logger. Because the other part of `if` statement is checking of log is winston Logger but not an actual instance of `Logger`